data-visualization/pages/02_Videos_and_channels.py

```diff
@@ -63,13 +111,11 @@ def add_expander_avg_values():
             st.warning(MSG_NO_DATA)
             return
 
-        col1, col2 = st.columns(2)
-        with col1:
+        st.markdown("Criteria to be included:")
```

Comment:
This text is properly displayed but I don't understand exactly its purpose.

![capture](https://user-images.githubusercontent.com/39056254/218424130-f057a6ae-8897-42a6-9703-cc176a9662ae.png)

When I read criteria I first thought about the rated criteria but it's something different right? If this text is a title to notify the presence of inputs that can filter the results, I think it can be safely deleted. The input looks explicit enough, and the UI will be more clear with less text here.

Reply:
Yes, at some point I put several criteria here, but then I move them in the sidebar so I can remove that

Reply:
I'll fix the label in another PR, don't lose to much time on dataviz, I think it is really not high priority.